coralproject/talk
# [CORL-1352] Bottom Links

## What does this PR do?

Fixed a style bug with the bottom links in the button to use the actual `iconLeft` prop.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



View the comment stream with the `ALTERNATE_OLDEST_FIRST_VIEW` feature flag. Sort by oldest first. Go to the bottom and see the link icons not underlined.
